genai-perf/tests/test_retrievers/test_synthetic_prompt_generator.py

```diff
@@ -28,24 +28,25 @@
 
 import pytest
 from genai_perf.config.input.config_command import ConfigCommand
+from genai_perf.config.input.config_defaults import TokenizerDefaults
 from genai_perf.exceptions import GenAIPerfException
 from genai_perf.inputs.retrievers.synthetic_prompt_generator import (
     SyntheticPromptGenerator,
 )
-from genai_perf.tokenizer import DEFAULT_TOKENIZER, get_tokenizer
+from genai_perf.tokenizer import get_tokenizer
 
 
 class TestSyntheticPromptGenerator:
 
     def test_synthetic_prompt_default(self):
         config = ConfigCommand({"model_name": "test_model"})
-        config.tokenizer.name = DEFAULT_TOKENIZER
+        config.tokenizer.name = "gpt2"
```

Comment:
does using gpt2 instead of llama change the numbers at all in the test?

Reply:
Not for these. It did for others, so I set the old default tokenizer to minimize the amount of change.